src/views/saphyr/SaphyrLibraries.vue

```diff
@@ -152,7 +151,7 @@ export default {
     // Get all the libraries
     // Provider function used by the bootstrap-vue table component
     async provider() {
-      this.items = await this.getMaterial(consts.MAT_TYPE_LIBRARIES)
+      this.items = await this.setLibraries()
```

Comment:
Maybe a more vuex'y way is for the provider to use call setLibraries(), then have a separate getter the get the libraries. PacbioRuns.vue does a similar thing

Reply:
Done. Also removed prefilter. Removed unneccessary getters.